src/app/components/tag/TagMenu.js

```diff
@@ -249,7 +249,7 @@ const TagMenuContainer = Relay.createContainer(withSetFlashMessage(TagMenuCompon
 // eslint-disable-next-line react/no-multi-comp
 class TagMenu extends React.PureComponent {
   render() {
-    const projectId = getCurrentProjectId(this.props.media.project_ids);
+    const projectId = getCurrentProjectId(this.props.media.project_id);
```

Comment:
Looks like the `getCurrentProjectId` and `getCurrentProject` helpers don't make sense anymore and can be removed.

Reply:
Removed both helpers